src/libponyc/ast/lexer.c

```diff
@@ -555,7 +556,10 @@ static void normalise_string(lexer_t* lexer)
       }
       else
       {
-        if(ws_this_line < ws)
+        if(!isspace(c)) {
+          has_non_ws = true;
+        }
```

Comment:
right, i still struggle to get my editor (and my head) to respect those formatting rules.